common/djangoapps/student/forms.py

```diff
@@ -242,6 +243,24 @@ def validate_name(name):
         raise forms.ValidationError(_('Full Name cannot contain the following characters: < >'))
 
 
+def validate_org_size(org_size):
+    possible_org_sizes = [os.code for os in TotalEmployee.objects.all()]
```

Comment:
`os` has been renamed.
These methods are not being used in multiple forms as of now, but since these are `student/forms` these have the potential of being used in more forms at a later point. Most of the validators in this file are not a part of any specific form.
Do you suggest that we make them a part of the class now, and maybe take out later when/if have to be re-used? Keeping in mind that it will cause substantial changes at that point. Let me know what you think. @saad-waqar

Reply:
make a file? for validators? give them not the identity they want but the identity they deserve